sessionctx/variable/sysvar.go

```diff
@@ -414,6 +414,12 @@ var defaultSysVars = []*SysVar{
 	{Scope: ScopeInstance, Name: PluginDir, Value: "/data/deploy/plugin", ReadOnly: true, GetGlobal: func(s *SessionVars) (string, error) {
 		return config.GetGlobalConfig().Instance.PluginDir, nil
 	}},
+	{Scope: ScopeInstance, Name: TiDBEnableConcurrencyDDL, Value: BoolToOnOff(DefTiDBEnableConcurrencyDDL), Hidden: true, Type: TypeBool, skipInit: true, SetGlobal: func(s *SessionVars, val string) error {
+		AllowConcurrencyDDL.Store(TiDBOptOn(val))
+		return nil
+	}, GetGlobal: func(s *SessionVars) (string, error) {
+		return BoolToOnOff(AllowConcurrencyDDL.Load()), nil
+	}},
```

Comment:
I'm curious why instance instead of GLOBAL? The advantage to GLOBAL is that it will persist - and no matching config item is being added here.

For `skipInit`: you can remove this. It will skip anywhere because there is no session scope (see https://github.com/pingcap/tidb/issues/35051 ). 

For `Hidden`: we have accepted a proposal to stop hiding sysvars. See: https://github.com/pingcap/tidb/blob/master/docs/design/2021-07-29-hidden-sysvars.md -- it causes problems for the support team.

Reply:
thank you morgo, it really helps me.